apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Switch } from "@thunderstore/cyberstorm";
+import React, { useState } from "react";
+
+const meta = {
+  title: "Cyberstorm/Components/Switch",
+  component: Switch,
+} as ComponentMeta<typeof Switch>;
+
+const defaultArgs = {
+  state: false,
+};
+
+const Template: ComponentStory<typeof Switch> = (args) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        gap: "0.25em",
+        backgroundColor: "#7e26b6",
+        padding: "1rem",
+      }}
+    >
+      <div>
+        <Switch {...args} />
+      </div>
+    </div>
+  );
+};
+
+const SameStateSwitchTemplate: ComponentStory<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  const switchCallback = (x: boolean) => {
```

Comment:
Seems like a rather pointless wrapper. Why not just use `setState` as the prop?

Reply:
Very true, it was pointless. Still not fully familiar, how data is passed to `onChange` function usually. (Radix passes the checked/unchecked bool in the `onCheckedChange`)